pkg/api/register.go

```diff
@@ -131,5 +132,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&metav1.APIGroup{},
 		&metav1.APIResourceList{},
 	)
+
+	metainternalversion.AddToGroupVersion(scheme, SchemeGroupVersion)
```

Comment:
@smarterclayton looks like we still have some meta type inconsistencies. The goal is that the internal types scheme can stand on its own feet, same for the v1 scheme.